### PR TITLE
docs(README): add FAQ and roadmap links

### LINKED ISSUE #9: Device (CSI) Local PV - GitHub updates
- [x] README Updates 
  - [x] Badges
  - [x] Project Status - Alpha
  - [x] k8s version compatibility
  - [x] Quickstart guide 
  - [x] Contributor Docs
  - [x] Adopters.md with links to openebs/openebs adopters
  - [x] Roadmap link to openebs project 
  - [x] Community Links 
- [ ] ~Helm Charts~
- [x] GitHub Builds
- [x] Multiarch builds
- [x] Disable Travis
- [ ] ~Downstream tagging~
- [x] e2e tests 
- [ ] ~Upgrades~
- [x] Monitoring
- [x] Troubleshooting guide

### PR DESCRIPTION
**Pull request by akhilerm**
Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ CSI Driver for using Local Block Devices
 
 ## Project Status
 
-Currently, the Device-LocalPV CSI Driver is in pre-alpha.
+Currently, the Device-LocalPV CSI Driver is in pre-alpha. The users/organizations using OpenEBS can be found [here](https://github.com/openebs/zfs-localpv/blob/master/Adopters.md)
```
Comment (pawanpraka1) [inline README.md]: zfs-localpv/blob/master/Adopters.md?

Comment (akhilerm) [reply README.md]: Sorry. Updated.

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Comment by pawanpraka1** [comment]
@akhilerm there seems to be some conflict. Can you resolve it and push the changes.

**Comment by akhilerm** [comment]
Done @pawanpraka1

**Pull request merged**